### PR TITLE
Android maps docs with Maplibre

### PR DESCRIPTION
**Pull request by jaspk06**
## What?

Adding missing docs for native Android maplibre

## Screenshots (optional)

Screen_Recording_20230801_150713_Getting Started_1

### CODE REVIEW

File: docs/maps/maps.mdx [hunk 1]
```diff
@@ -397,6 +397,176 @@ class ViewController: UIViewController, MGLMapViewDelegate {
 }
 ```
 
+### Android
+
+Radar maps can be integrated into an Android application with [MapLibre Native](https://github.com/maplibre/maplibre-native).
+
+To get started, add the MapLibre Native dependency into your module Gradle file (usually located at `<project>/<app-module>/build.gradle`). Replace `<version>` with the latest MapLibre Native version (e.g.: `org.maplibre.gl:android-sdk:10.2.0`).
+
+```kotlin title="build.gradle"
```
Comment (palmakkar) [inline docs/maps/maps.mdx:406]: I like these file names for clarity, but given we don't use them anywhere else in docs yet let's just leave it out for consistency

File: docs/maps/maps.mdx [hunk 2]
```diff
@@ -397,6 +397,176 @@ class ViewController: UIViewController, MGLMapViewDelegate {
...
+            .fromBitmap(bitmapMarker)
+
+        // create point to place marker
+        val markerOptions = MarkerOptions()
+            .position(coordinate)
+            .icon(icon)
+        mapboxMap.addMarker(markerOptions)
+    }
+
+    override fun onStart() {
```
Comment (palmakkar) [inline docs/maps/maps.mdx:533]: should we leave out any boilerplate code and keep the snippets relevant to this step? think that would be more consistent with the way we've documented the other platforms

File: docs/maps/maps.mdx (hunk 1 above)
Comment (jaspk06) [reply docs/maps/maps.mdx:406]: the issue with android is that there's multiple config files with the same name like `build.gradle` where it appears in 3 different directories so I feel like it still needs to be included somehow. I agree with the other code blocks tho, the file names are unnecessary

File: docs/maps/maps.mdx (hunk 2 above)
Comment (jaspk06) [reply docs/maps/maps.mdx:533]: the boilerplate is kinda necessary in this case cuz on each lifecycle call like `onStart()` `mapView.onStart()` also needs to be called.

Comment (jaspk06) [reply docs/maps/maps.mdx:533]: which isn't already included in the default boilerplate

**Review by Gdakgdak22 — APPROVED** [review]

**Comment by lmeier** [comment]
FYI we're using this map logo on React Native, per @berylw's instruction: 
image

https://github.com/radarlabs/react-native-radar/blob/master/js/ui/map-logo.png

**Comment by kochis** [comment]
I think we should be using the same maps logo everywhere for consistency. The darker one should be the correct one.

**Review by palmakkar — APPROVED** [review]

**Review by Gdakgdak22 — APPROVED** [review]

**Pull request merged**